project.clj

```diff
@@ -10,10 +10,9 @@
                  [re-frame "1.2.0"]
                  ;; handlerForForeign
                  [com.cognitect/transit-cljs "0.8.256"]
-                 ;; web communications (fork is used because of https://github.com/ptaoussanis/sente/pull/357)
-                 [re-frisk/sente "1.15.0"]
+                 [re-frisk/sente "1.20.0"]
```

Comment:
this change is strange

Reply:
Yes, this is a mistake. Should be `[com.taoensso/sente "1.20.0"]` (latest version of sente includes the react native fix you included as a patch in your fork)